app/js/tabmanager.js

```diff
@@ -225,9 +227,11 @@ const TabManager = {
    * @param {mixed} tabId
    *  Tab ID or Tab object.
    */
-  updateLastAccessed(tabOrTabId: chrome$Tab | number) {
+  updateLastAccessed(tabOrTabId: TabOrTabId | Array<TabOrTabId>) {
     let tabId;
-    if (typeof tabOrTabId === 'number') {
+    if (Array.isArray(tabOrTabId)) {
+      tabOrTabId.map(this.updateLastAccessed.bind(this));
```

Comment:
This case should immediately `return;`. This case knows tabId is a not a number, so it doesn't need to call `console.log` and report an error on line 241 below.

Reply:
It's fixed now.

Reply:
Excellent, thanks again for this PR.